(window.webpackJsonp=window.webpackJsonp||[]).push([[20],{348:function(t,a,e){"use strict";e.r(a);var s=e(4),r=Object(s.a)({},(function(){var t=this,a=t.$createElement,e=t._self._c||a;return e("ContentSlotsDistributor",{attrs:{"slot-key":t.$parent.slotKey}},[e("p",[t._v("题目: 用于转录分布预测和细胞类型反卷积的空间和单细胞转录组学整合方法的基准测试"),e("br"),t._v("\nDOI: "),e("a",{attrs:{href:"https://doi.org/10.1038/s41592-022-01480-9",target:"_blank",rel:"noopener noreferrer"}},[t._v("https://doi.org/10.1038/s41592-022-01480-9"),e("OutboundLink")],1),e("br"),t._v("\nCite: Li, B., Zhang, W., Guo, C. et al. Benchmarking spatial and single-cell transcriptomics integration methods for transcript distribution prediction and cell type deconvolution. Nat Methods 19, 662–670 (2022).")]),t._v(" "),e("p",[e("strong",[t._v("作者介绍：")])]),t._v(" "),e("table",[e("thead",[e("tr",[e("th",{staticStyle:{"text-align":"center"}},[t._v("Kun Qu")])])]),t._v(" "),e("tbody",[e("tr",[e("td",{staticStyle:{"text-align":"center"}},[e("img",{attrs:{src:"http://jcyx.ustc.edu.cn/_upload/article/images/98/e7/b5f2b7524ecc95a13264a819b160/f8d7a581-a9a7-4eab-8e04-36ab469e3dad.png",width:"200"}})])]),t._v(" "),e("tr",[e("td",{staticStyle:{"text-align":"center"}},[t._v("中国科学技术大学")])]),t._v(" "),e("tr",[e("td",{staticStyle:{"text-align":"center"}},[e("a",{attrs:{href:"mailto:qukun@ustc.edu.cn"}},[t._v("qukun@ustc.edu.cn")])])])])]),t._v(" "),e("h2",{attrs:{id:"abstract"}},[e("a",{staticClass:"header-anchor",attrs:{href:"#abstract"}},[t._v("#")]),t._v(" Abstract:")]),t._v(" "),e("p",[t._v("空间转录组学方法极大地提高了我们检测组织中RNA转录本空间分布的能力，但表征空间中单细胞的全转录组水平数据仍然具有挑战性。\n为了满足这一需求，研究人员开发了整合方法，将空间转录组数据与单细胞 RNA-seq 数据相结合，以预测未检测到的转录本的空间分布和/或对组织学切片中的 spot 进行细胞类型反卷积。\n然而，迄今为止，还没有独立研究对这些集成方法进行比较分析以对其性能进行基准测试。\n在这里，我们使用 45 个配对数据集（包括空间转录组学和 scRNA-seq 数据）和 32 个模拟数据集对 16 种集成方法进行基准测试。\n我们发现"),e("span",{staticStyle:{color:"red"}},[t._v("Tangram、gimVI、SpaGE")]),t._v("在预测RNA转录本空间分布方面优于其他集成方法，而"),e("span",{staticStyle:{color:"red"}},[t._v("Cell2location、SpatialDWLS、RCTD")]),t._v("是spot细胞类型反卷积的最佳方法。\n我们提供基准管道来帮助研究人员选择最佳的集成方法来处理他们的数据集。")]),t._v(" "),e("h2",{attrs:{id:"results"}},[e("a",{staticClass:"header-anchor",attrs:{href:"#results"}},[t._v("#")]),t._v(" Results:")]),t._v(" "),e("h3",{attrs:{id:"figure-1-benchmarking-workflow-and-summary-characteristics-of-the-examined-paired-datasets"}},[e("a",{staticClass:"header-anchor",attrs:{href:"#figure-1-benchmarking-workflow-and-summary-characteristics-of-the-examined-paired-datasets"}},[t._v("#")]),t._v(" Figure 1: Benchmarking workflow and summary characteristics of the examined paired datasets.")]),t._v(" "),e("p",[e("img",{attrs:{src:"https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41592-022-01480-9/MediaObjects/41592_2022_1480_Fig1_HTML.png?as=webp",alt:"Figure 1"}})]),t._v(" "),e("div",{staticClass:"custom-block note"},[e("p",{staticClass:"custom-block-title"},[t._v("Figure 1. 基准测试工作流程和所检查的配对数据集的摘要特征")]),t._v(" "),e("p",[e("strong",[t._v("(a)")]),t._v(" 用于比较配对空间转录组学和 scRNA-seq 数据集的集成方法性能的基准测试工作流程的示意图。我们使用 16 种整合方法来组合空间和单细胞转录组数据，然后比较它们在 (1) 预测 RNA 转录本的空间分布和 (2) 组织学 spot 的细胞类型去卷积方面的性能。我们还评估了集成方法消耗的计算资源。"),e("br"),t._v(" "),e("strong",[t._v("(b)")]),t._v(" 本研究中使用的 45 个配对数据集和 32 个模拟数据集的信息：每个数据集包含同一组织的空间转录组数据和 scRNA-seq 数据。")])]),t._v(" "),e("h3",{attrs:{id:"figure-2-comparing-the-accuracy-of-eight-integration-methods-capable-of-predicting-the-spatial-distribution-of-rna-transcripts"}},[e("a",{staticClass:"header-anchor",attrs:{href:"#figure-2-comparing-the-accuracy-of-eight-integration-methods-capable-of-predicting-the-spatial-distribution-of-rna-transcripts"}},[t._v("#")]),t._v(" Figure 2: Comparing the accuracy of eight integration methods capable of predicting the spatial distribution of RNA transcripts.")]),t._v(" "),e("p",[e("img",{attrs:{src:"https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41592-022-01480-9/MediaObjects/41592_2022_1480_Fig2_HTML.png?as=webp",alt:"Figure 2"}})]),t._v(" "),e("div",{staticClass:"custom-block note"},[e("p",{staticClass:"custom-block-title"},[t._v("Figure 2. 比较八种能够预测 RNA 转录本空间分布的整合方法的准确性")]),t._v(" "),e("p",[e("strong",[t._v("(a,b)")]),t._v(" 数据集 4（seqFISH+；Smart-seq；小鼠皮质）中Igsf21 ( a ) 和Rprm ( b )的空间分布，包括真实值和每种集成方法的预测结果。PCC，真实转录本的表达向量与预测结果的表达向量之间的皮尔逊相关系数。"),e("br"),t._v(" "),e("strong",[t._v("(c)")]),t._v(" 每种集成方法预测数据集4中转录本空间分布的PCC、SSIM、RMSE和JS条形图。数据以平均值±95%置信区间表示。"),e("br"),t._v(" "),e("strong",[t._v("(d)")]),t._v(" 数据集 4 中转录本的 8 种整合方法的 AS 小提琴图(从 PCC、SSIM、RMSE 和 JS 值聚合；参见方法)；n = 4 个基准指标。"),e("br"),t._v(" "),e("strong",[t._v("(e)")]),t._v(" 所有 45 个配对数据集的 8 种整合方法的 AS 箱线图。n = 45 个独立数据集。")])]),t._v(" "),e("h3",{attrs:{id:"figure-3-comparing-the-accuracy-of-the-eight-integration-methods-for-sparse-spatial-expression-matrices-down-sampled-from-the-original-datasets-using-splatter"}},[e("a",{staticClass:"header-anchor",attrs:{href:"#figure-3-comparing-the-accuracy-of-the-eight-integration-methods-for-sparse-spatial-expression-matrices-down-sampled-from-the-original-datasets-using-splatter"}},[t._v("#")]),t._v(" Figure 3: Comparing the accuracy of the eight integration methods for sparse spatial expression matrices down-sampled from the original datasets using Splatter.")]),t._v(" "),e("p",[e("img",{attrs:{src:"https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41592-022-01480-9/MediaObjects/41592_2022_1480_Fig3_HTML.png?as=webp",alt:"Figure 3"}})]),t._v(" "),e("div",{staticClass:"custom-block note"},[e("p",{staticClass:"custom-block-title"},[t._v("Figure 3. 比较使用 Splatter 从原始数据集下采样的稀疏空间表达矩阵的八种整合方法的准确性")]),t._v(" "),e("p",[e("strong",[t._v("(a)")]),t._v(" 根据原始数据和数据集 4 的下采样数据预测的转录本的空间分布的 PCC。对于原始数据和下采样数据，红色转录本的 PCC 值都大于 0.5。红色转录本在所有转录本中的比例定义为RS。"),e("br"),t._v(" "),e("strong",[t._v("(b)")]),t._v(" 8种整合方法以不同下采样率进行下采样在处理数据集 4 的稀疏表达矩阵时的RS值。"),e("br"),t._v(" "),e("strong",[t._v("(c)")]),t._v(" 处理下采样数据集的稀疏表达矩阵时八种整合方法的RS值。n = 19 个独立数据集。")])]),t._v(" "),e("h3",{attrs:{id:"figure-4-comparing-the-performance-of-the-12-integration-methods-capable-of-deconvoluting-cell-types-of-each-histological-spot"}},[e("a",{staticClass:"header-anchor",attrs:{href:"#figure-4-comparing-the-performance-of-the-12-integration-methods-capable-of-deconvoluting-cell-types-of-each-histological-spot"}},[t._v("#")]),t._v(" Figure 4: Comparing the performance of the 12 integration methods capable of deconvoluting cell types of each histological spot.")]),t._v(" "),e("p",[e("img",{attrs:{src:"https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41592-022-01480-9/MediaObjects/41592_2022_1480_Fig4_HTML.png?as=webp",alt:"Figure 4"}})]),t._v(" "),e("div",{staticClass:"custom-block note"},[e("p",{staticClass:"custom-block-title"},[t._v("Figure 4. 比较能够对每个组织学点的细胞类型进行解卷积的 12 种整合方法的性能")]),t._v(" "),e("p",[e("strong",[t._v("(a)")]),t._v(" 数据集 10 的 STARmap 切片（STARmap; Smart-seq; mouse visual cortex），其中细胞按细胞类型注释。每个网格代表一个包含多个单元的模拟点。"),e("br"),t._v(" "),e("strong",[t._v("(b)")]),t._v(" 8种整合方法以不同下采样率进行下采样在处理数据集 4 的稀疏表达矩阵时的RS值。"),e("br"),t._v(" "),e("strong",[t._v("(c,d)")]),t._v(" 从数据集 10 ( c ) 和数据集 4 ( d )模拟的组织学spot的细胞类型组成的AS 条形图。"),e("br"),t._v(" "),e("strong",[t._v("(e)")]),t._v(" 所有 32 个模拟数据集的 10 种集成方法的 AS 箱线图。SpaOTsc 和 novoSpaRc 被排除在外，因为它们需要每个点的空间位置信息，而这在模拟数据集中不可用。")])])])}),[],!1,null,null,null);a.default=r.exports}}]);