(window.webpackJsonp=window.webpackJsonp||[]).push([[27],{356:function(t,e,r){"use strict";r.r(e);var i=r(4),a=Object(i.a)({},(function(){var t=this,e=t.$createElement,r=t._self._c||e;return r("ContentSlotsDistributor",{attrs:{"slot-key":t.$parent.slotKey}},[r("p",[t._v("题目: chromVAR：从单细胞表观基因组数据推断转录因子相关的可及性"),r("br"),t._v("\nDOI: "),r("a",{attrs:{href:"https://doi.org/10.1038/nmeth.4401",target:"_blank",rel:"noopener noreferrer"}},[t._v("https://doi.org/10.1038/nmeth.4401"),r("OutboundLink")],1),r("br"),t._v("\nCite: Schep, A., Wu, B., Buenrostro, J. et al. chromVAR: inferring transcription-factor-associated accessibility from single-cell epigenomic data. Nat Methods 14, 975–978 (2017).")]),t._v(" "),r("p",[r("strong",[t._v("作者介绍：")])]),t._v(" "),r("table",[r("thead",[r("tr",[r("th",{staticStyle:{"text-align":"center"}},[t._v("William J Greenleaf")])])]),t._v(" "),r("tbody",[r("tr",[r("td",{staticStyle:{"text-align":"center"}},[r("img",{attrs:{src:"https://med.stanford.edu/services/api/cap/profiles/photocache.24604.jpg",width:"200"}})])]),t._v(" "),r("tr",[r("td",{staticStyle:{"text-align":"center"}},[t._v("Department of Genetics, Stanford University School of Medicine, Stanford, California, USA")])]),t._v(" "),r("tr",[r("td",{staticStyle:{"text-align":"center"}},[r("a",{attrs:{href:"mailto:wjg@stanford.edu"}},[t._v("wjg@stanford.edu")])])])])]),t._v(" "),r("h2",{attrs:{id:"abstract"}},[r("a",{staticClass:"header-anchor",attrs:{href:"#abstract"}},[t._v("#")]),t._v(" Abstract:")]),t._v(" "),r("p",[t._v("单细胞 ATAC-seq (scATAC) 产生的数据稀疏，这给传统分析带来了挑战。我们开发了 chromVAR "),r("a",{attrs:{href:"http://www.github.com/GreenleafLab/chromVAR",target:"_blank",rel:"noopener noreferrer"}},[t._v("http://www.github.com/GreenleafLab/chromVAR"),r("OutboundLink")],1),t._v("，这是一个 R 包，用于通过估计共享相同motif或注释的peaks内可及性的增益或损失来分析稀疏染色质可及性数据，同时控制技术偏差。chromVAR 能够准确聚类 scATAC-seq 图谱，并表征与染色质可及性变化相关的已知和de novo sequence motifs。")]),t._v(" "),r("h2",{attrs:{id:"main"}},[r("a",{staticClass:"header-anchor",attrs:{href:"#main"}},[t._v("#")]),t._v(" Main:")]),t._v(" "),r("h3",{attrs:{id:"figure-1-chromvar-enables-interpretable-analysis-of-sparse-chromatin-accessibility-data"}},[r("a",{staticClass:"header-anchor",attrs:{href:"#figure-1-chromvar-enables-interpretable-analysis-of-sparse-chromatin-accessibility-data"}},[t._v("#")]),t._v(" Figure 1: chromVAR enables interpretable analysis of sparse chromatin-accessibility data.")]),t._v(" "),r("p",[r("img",{attrs:{src:"https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fnmeth.4401/MediaObjects/41592_2017_BFnmeth4401_Fig1_HTML.jpg?as=webp",alt:"Figure 1"}})]),t._v(" "),r("div",{staticClass:"custom-block note"},[r("p",{staticClass:"custom-block-title"},[t._v("Figure 1. chromVAR 能够对稀疏染色质可及性数据进行可解释的分析")]),t._v(" "),r("p",[r("strong",[t._v("(a)")]),t._v(" chromVAR 聚合具有共同特征（例如 motif ）的 peaks 上的染色质可及性，并应用偏差校正来生成可用于下游分析的每个细胞或样本的分数。"),r("br"),t._v(" "),r("strong",[t._v("(b)")]),t._v(" 对来自不同造血群体的 77 个样本在对完整数据的总测序读数进行下采样之前和之后的偏倚校正偏差的 Pearson 相关性。"),r("br"),t._v(" "),r("strong",[t._v("(c)")]),t._v(" 不同样品的 tSNE 可视化，使用根据每个样品下采样到 10,000 个片段（单个细胞的典型数量）的数据计算出的归一化偏差。")])]),t._v(" "),r("h3",{attrs:{id:"figure-2-chromvar-can-be-used-to-cluster-single-cells-and-interpret-motifs-underlying-chromatin-accessibility-variation"}},[r("a",{staticClass:"header-anchor",attrs:{href:"#figure-2-chromvar-can-be-used-to-cluster-single-cells-and-interpret-motifs-underlying-chromatin-accessibility-variation"}},[t._v("#")]),t._v(" Figure 2: chromVAR can be used to cluster single cells and interpret motifs underlying chromatin-accessibility variation.")]),t._v(" "),r("p",[r("img",{attrs:{src:"https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fnmeth.4401/MediaObjects/41592_2017_BFnmeth4401_Fig2_HTML.jpg?as=webp",alt:"Figure 2"}})]),t._v(" "),r("div",{staticClass:"custom-block note"},[r("p",{staticClass:"custom-block-title"},[t._v("Figure 2. chromVAR 可用于对单细胞进行聚类并解释染色质可及性变异背后的 motif")]),t._v(" "),r("p",[r("strong",[t._v("(a)")]),t._v(" 基于 chromVAR 原始（左）或偏差校正偏差（右）的 1,561 个单细胞的 tSNE 可视化。"),r("br"),t._v(" "),r("strong",[t._v("(b)")]),t._v(" 火山图显示 AML 母细胞 ( n = 122) 和 LSC 细胞 ( n = 144 ) 之间每个 motif 的偏差校正可及性偏差 (左) 和变异性 (右) 的平均差异。"),r("br"),t._v(" "),r("strong",[t._v("(c)")]),t._v(" 不同簇中选定标记基因的基因得分的空间映射和选定基因的染色质可及性具有高度组织特异性。")])]),t._v(" "),r("h3",{attrs:{id:"figure-3-chromvar-identifies-de-novo-motifs-associated-with-chromatin-accessibility-variation-in-single-cells"}},[r("a",{staticClass:"header-anchor",attrs:{href:"#figure-3-chromvar-identifies-de-novo-motifs-associated-with-chromatin-accessibility-variation-in-single-cells"}},[t._v("#")]),t._v(" Figure 3: chromVAR identifies de novo motifs associated with chromatin-accessibility variation in single cells.")]),t._v(" "),r("p",[r("img",{attrs:{src:"https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fnmeth.4401/MediaObjects/41592_2017_BFnmeth4401_Fig3_HTML.jpg?as=webp",alt:"Figure 3"}})]),t._v(" "),r("div",{staticClass:"custom-block note"},[r("p",{staticClass:"custom-block-title"},[t._v("Figure 3. chromVAR从头识别与单细胞染色质可及性变异相关的 motif")]),t._v(" "),r("p",[r("strong",[t._v("(a)")]),t._v(" 基于细胞间标准化偏差向量的 motif 和 k-mers 之间相似性的 tSNE 可视化。"),r("br"),t._v(" "),r("strong",[t._v("(b)")]),t._v(" 与种子 k-mer AGATAAG 存在一处不匹配的 k-mer 的共享变异性。"),r("br"),t._v(" "),r("strong",[t._v("(c)")]),t._v(" chromVAR 使用七聚体偏差分数组装的de novo motif 示例，最接近的匹配已知 motif 紧接在下面。"),r("br"),t._v(" "),r("strong",[t._v("(d)")]),t._v(" c中每对的de novo motif 和已知 motif 的变异性（ z分数的 sd ）。"),r("br"),t._v(" "),r("strong",[t._v("(e)")]),t._v(" de novo motif 和c中已知 motif 之间的 motif 相似性评分。"),r("br"),t._v(" "),r("strong",[t._v("(f)")]),t._v(" c中每对de novo motif 和已知 motif 的标准化偏差之间的 Pearson 相关性。")])])])}),[],!1,null,null,null);e.default=a.exports}}]);